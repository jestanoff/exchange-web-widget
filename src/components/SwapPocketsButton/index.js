import React from 'react';

const SwapPocketsButton = () => (<span>Swap</span>)

export default SwapPocketsButton;
