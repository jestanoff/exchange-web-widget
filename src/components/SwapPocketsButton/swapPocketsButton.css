.button {
  background: #ffffff;
  border-radius: 50%;
  border: 1px solid  #f0f0f0;
  height: 16px;
  margin-left: 36px;
  padding: 10px;
  text-align: center;
  width: 16px;
}

.icon {
  fill: #0b5ce4;
  height: 16px;
  width: 16px;
}
