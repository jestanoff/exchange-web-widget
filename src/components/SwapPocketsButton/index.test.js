import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import SwapPocketsButton from './';

describe('SwapPocketsButton component', () => {
  test('should render proper markup', () => {
    const { container } = render(<SwapPocketsButton />);

    expect(container).toMatchSnapshot();
  });
});
