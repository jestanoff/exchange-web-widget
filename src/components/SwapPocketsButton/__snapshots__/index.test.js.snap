// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SwapPocketsButton component should render proper markup 1`] = `
<div>
  <span>
    Swap
  </span>
</div>
`;
